README.md

```diff
@@ -81,6 +81,7 @@ Plugin supports special global variables which are allow to change behaviour of
  - **`g:cmake_compile_commands_link`** set the path for a link on compile_commands.json. Default is empty.
  - **`g:cmake_build_executor`** allows to force set the build executor. Available values are 'job', 'dispatch', 'system' and ''. Default is empty.
  - **`g:cmake_vimspector_support`** enables generation and modification of [Vimspector](https://github.com/puremourning/vimspector) config file. Default is 0. **Attention! The support of Vimspector config is an experimental feature.**
+ - **`g:cmake_variants`** enables predefined cmake build variants in the form of a dictionary, e.g. `{ 'Debug' : { 'cmake_build_type' : 'Debug', 'cmake_usr_args' : '-DCONAN_PATH=~/.conan' } }`
```

Comment:
I think we cannot change the cmake_build_type for default targets. Maybe we need other example.

Reply:
I wanted to enable to user to override default build types and add custom ones.
```
{
    'Debug' : { 'cmake_build_type' : 'Debug', 'cmake_usr_args' : '-blahblah' }
}
```
... would override the default `Debug` build, but `Release, RelWithDebInfo, MinSizeRel` still have the default values specified by `g:cmake_usr_args` and `g:cmake_build_type`.

Because if someone wants to have custom `debug`, `release`, `minSizeRel`, why not allow them to override the existing ones, instead of having 3 they will never use which are similarly named as the 3 custom ones.